atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -138,6 +143,21 @@ def get_next_release_append(self, component, version, base_release, base_suffix=
                     return next_release
             suffix += 1
 
+    def get_next_release(self, build_info):
+        queryopts = {'order': '-build.id', 'limit': 1}
+        # release is either single number or with decimal point, so we can easily bump next release
+        # pylint: disable=W1401
+        search_str = '^{}-{}-\d+(\.\d+)?$'.format(build_info['name'], build_info['version'])
```

Comment:
Is there a reason not to use an r-string here? That should get rid of the pylint warning.

Reply:
added 'r'